pdp/policy.go

```diff
@@ -243,6 +250,42 @@ func (p *Policy) delChild(ID string) (*Policy, error) {
 	return p.updatedCopy(rules, algorithm), nil
 }
 
+// DepthMarshal implements StorageMarshal
+func (p Policy) DepthMarshal(out io.Writer, depth int) error {
+	if depth < 0 {
+		return fmt.Errorf("depth must be >= 0, got %d", depth)
+	}
+	if p.hidden {
+		return errHiddenPolicy
```

Comment:
Don't think we want to fail on hidden policy. It's better to show it or silently skip. To choose behavior we can add special request option like `/storage/x/y/z?show=all` later. For now it's simpler to not skip hidden policies.